add documentation on using checksum (#18)
MeteoSwiss/opendata-forecast-data: README.md

```
wget -O <desired_filename> “<pre-signed URL>”
```

> After the download you may want to check your forecast data's integrity.

Reviewer: Tried to reformulate for "the download" and "you may want"
```suggestion
After downloading your forecast data, it's good practice to verify its integrity before use.
```

<details>

> <summary> Verify Data Integrity</summary>

Reviewer: This info seems as important as the others—maybe don’t label it a 'detail'?

> To ensure the downloaded file is not corrupted, generate its SHA-256 hash value and compare it to the official checksum provided within the `curl` tool.

Reviewer: - tried to reformulate "official checksum" by describing that it's the one included in the file's header field.

- I removed "curl" because it's only the tool used to fetch the information

```suggestion
To ensure the downloaded file is not corrupted, compute its SHA-256 hash and verify it against the checksum provided in the file's header field.
```

> 

Reviewer: ```suggestion

**Steps:**
```

**Steps:**

> 1. Open a Terminal and generate the SHA-256 checksum of the downloaded file:

Reviewer: ```suggestion
1. Open a terminal and generate the SHA-256 checksum of the downloaded file:
```

> 1. Open a Terminal and run the following command to generate the checksum of the downloaded file:

Reviewer: shorten
```suggestion
1. Generate the SHA-256 checksum of the downloaded file:
```

sha256sum <downloaded_filename>
```

> 2. Use the following command to retrieve the official checksum:
> ```
> curl -i <pre-signed URL>
> ```
> 3. After a successful request the provided checksum is located under the  `x-amz-meta-sha256` key.

Reviewer: In step 2, checksum isn't retrieved here. We can merge step 2 and 3 in one command. 
The quotes on the pre-signed url need to be added
````suggestion
2. Retrieve the checksum from the file’s header field `x-amz-meta-sha256` using the following command:
```
curl -s -i "<pre-signed URL>" | awk -F': ' '/x-amz-meta-sha256/ {print $0}'
```
````

> 4. Compare both hash values. If they match, the forecast data is save to use.

Reviewer: typo
```suggestion
3. Compare the two hash values. If they match, your forecast data file is safe to use.
```

</details>

<br />

> Once downloaded and verified, proceed with decoding the GRIB file using the instructions in [Section 2.7.4 Decoding GRIB Files with ecCodes](#274-decoding-grib-files-with-eccodes).

Reviewer: adding "the file"
```suggestion
Once the file is verified, you can proceed with decoding the GRIB file using the instructions in [Section 2.7.4 Decoding GRIB Files with ecCodes](#274-decoding-grib-files-with-eccodes).
```
